spec/support/locale_lap.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+module LocaleLabTestHelper
+  def self.restore_backup
+    @original_i18n_files.each do |file|
+      File.write(file[:path], file[:content])
+    end
+  end
+
+  def self.take_backup
+    @original_i18n_files = LocaleLab::TranslationFile.all.map do |file|
+      { path: file.path, content: file.to_yaml }
+    end
+  end
+end
+
+RSpec.configure do |config|
+  config.before(:suite) do
+    LocaleLabTestHelper.take_backup
+  end
+
+  config.before(:each) do
+    Thread.current[:locale_lab] = nil
+  end
+
+  config.after(:each) do
+    LocaleLabTestHelper.restore_backup
+  end
```

Comment:
```suggestion
  config.before(:suite) do
    LocaleLabTestHelper.take_backup
  end

  config.before(:each) do
    LocaleLabTestHelper.restore_backup
    Thread.current[:locale_lab] = nil
  end

  config.after(:suite) do
    LocaleLabTestHelper.restore_backup
  end
```

Reply:
Wellicht is het beter om de bestanden die eigenlijk nooit mogen wijzigen door het runnen van een spec op een andere plek te zetten (bijv. de spec/fixtures/ folder) en die met een `before(:suite)` te kopiëren naar de plek waar ze moeten staan. 

Dit zorgt ervoor dat we altijd 100% zeker weten dat de bestanden die bewerkt worden tijdens het runnen van de spec suite nooit permanent gewijzigd zijn tussen spec runs in (bijv. als je de computer uitzet tijdens het draaien van een individuele spec die wijzigingen heeft gemaakt).